index.html

```diff
@@ -236,29 +236,25 @@ <h3>Stoica Robert-Cristian</h3>
 							<div class="custom-card">
 								<div class="avatar-wrapper">
 									<img
-										src="https://media.istockphoto.com/vectors/thumbnail-image-vector-graphic-vector-id1147544807?k=6&m=1147544807&s=612x612&w=0&h=8CXEtGfDlt7oFx7UyEZClHojvDjZR91U-mAU8UlFF4Y="
+										src="https://i.ibb.co/TLhMXMX/326520486-513017780710409-1049115559909250766-n.jpg"
 										alt=""
```

Comment:
Vad ca ai salvat si poza local, te-as sfatui sa muti poza in folder-ul assets si sa redenumesti poate in 'cristian-petrisor.jpg', si apoi aici sa o incarci ii schimbi src-ul:
`<img src="assets/cristian-petrisor.jpg" alt="" class="avatar"/>`